routes/web.php

```diff
@@ -240,6 +240,10 @@
         });
     });
 
+    if (Features::isEnabled('hxl')) {
```

Comment:
@rowasc added this to demo how we could toggle entire routes based on a feature

Reply:
yay thank you